obs-server/trigger_clang_build

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+PKG=$1
+
+cd /root
+if [ ! -d Debian\:Unstable\:Clang ]; then
+osc co Debian:Unstable:Clang > /dev/null 2>&1
+fi
+cd Debian\:Unstable\:Clang/
+osc up > /dev/null 2>&1
+
+if [ "$PKG" == "obs-service-clang-build" ]; then
+  if ! osc ls Debian:Unstable:Clang obs-service-clang-build > /dev/null 2>&1; then
+    echo 'Preparing obs-service-clang-build package...'
+    mkdir obs-service-clang-build
+    cd obs-service-clang-build/
+    wget 'https://www.ime.usp.br/~athoscr/files/obs/obs-service-clang-build_0.1-1.debian.tar.xz'
```

Comment:
please add this to this repo and use that. I would like to avoid external dependencies

Reply:
yeah, better than on thirdparty website.

Reply:
Done.